etl/steps/data/garden/countries/2023-09-25/isd.meta.yml

```diff
@@ -4,12 +4,33 @@
 # Learn more about the available fields:
 # http://docs.owid.io/projects/etl/en/latest/architecture/metadata/reference/dataset/
 dataset:
+  title: States in regions (ISD)
   update_period_days: 1400
 
 
 # Learn more about the available fields:
 # http://docs.owid.io/projects/etl/en/latest/architecture/metadata/reference/tables/
 tables:
+  isd_regions:
+    title: States in regions (ISD)
+    variables:
+      number_countries:
+        title: Number of states in a region
+        description_short: |-
+          Number of sovereign states in each region.
+        unit: countries
+        processing_level: minor
+        description_processing: |-
+          We assign each sovereign state to a region based on the mapping (using ISD/COW codes):
+
+            - Americas: 2-199
+            - Europe: 200-399, 2558, 3375
+            - Africa: 400-626, 4044-6257
+            - Middle East: 630-699, 6821-6845
+            - Asia and Oceania: 700-999, 7003-9210
```

Comment:
These ids don't coincide with the ones used in the script

Reply:
Thanks for the catch! This should match now (except for codes 75911, 85421, which are custom made so I decided not to include them here).